_pages/about.md

```diff
@@ -36,35 +36,10 @@ Get my latest resume from [here](https://praphulsamavedam.github.io/resume/)
 Have a look at [my cv](https://praphulsamavedam.github.io/cv/)    
 
 ## Updates:
-This website is under development so some links might have placeholders, so kindly bare with me , else :raised_hand: via 
```

Comment:
Home page changes addressing the lack of spaces, and fixes the broken images links.